extensions/csv/src/main/java/io/deephaven/csv/densestorage/QueueReader.java

```diff
@@ -0,0 +1,211 @@
+package io.deephaven.csv.densestorage;
+
+import io.deephaven.csv.containers.ByteSlice;
+import io.deephaven.csv.containers.CharSlice;
+
+public class QueueReader<TARRAY> {
```

Comment:
Class javadoc?

Reply:
Alternatively, if these are implementation details it would be better to make them non-public IMO

Reply:
Can you clarify whether you mean the class should not be public (it should be "internal" instead), or that you want me to write all the getters and setters for the fields.

I suppose having getters/setters is more "Java Beans" style, but I find it sooooo tedious.

Of course, I will comply with whatever you think the proper style is here.

Reply:
I think the spirit is this: If a class is public, it needs a javadoc.  If a class is an implementation detail, it doesn't have to be public, and it may or may not have javadoc (sometimes javadoc is the best way to document something, so if you need to write comments, those comments are better off as javadoc for package private classes).   Less public classes is better, so what doesn't need to be public probably shouldn't be.